### PR TITLE
Added files related to slider component --STALE PR--

### PR DESCRIPTION
**Pull request by judzelicor**
- Slider Color scheme works for dark and light mode.
- Mobile responsive but if you don't feel satisfied please tell on how I should improve it.
- If I'm breaking any clean code conventions and whatnot (especially on my css) please do tell me.
- I'll clean up the code for the slider right after your feedback.
- Any input welcome!!!

### CODE REVIEW

**Comment by judzelicor** [comment]
Huh, I just visited the preview and somehow the controls for the slider are positioned way at the top however in my environment it's inside the slider.

**Comment by judzelicor** [comment]
Oh wait I found my mistake  that's causing this.

File: public/test.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <li class="sliderItem">Card 9</li>
+    </ul>
+    <div class="controls">
+      <div class="leftControl">left</div>
+      <div class="rightControl">right</div>
+    </div>
+  </div>
+  <script src="./components/carousel/carousel.js"></script>
+</body>
+</html>
```
Comment (zacharyselk) [inline public/test.html:60]: Remove this file.

File: public/components/carousel/slider.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    this.sliderContainer.children[this.prevCardIndex].classList.add('prev')
+    this.sliderContainer.children[this.nextCardIndex].classList.add('next')
+  }
+}
+
+const sliders = document.querySelectorAll('.slider')
+
+for (let i = 0; i < sliders.length; i++) {
+  const HTMLSlider = sliders[i]
+  console.log(HTMLSlider)
```
Comment (zacharyselk) [inline public/components/carousel/slider.js:75]: Remove unneeded logging

File: public/components/carousel/slider.css [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+.controls {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 40px;
+  position: absolute;
+  top: 50%;
+  width: 100vw;
+  z-index: 999;
+}
```
Comment (zacharyselk) [inline public/components/carousel/slider.css:102]: Currently this flex-box stretches across the sliderContainer and covers anything that appears behind it. This is causes most of the button  in the cards to be covered and not receive to `hover` or `click` events.

**Review by zacharyselk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**